pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/user/UserDao.java

```diff
@@ -65,4 +69,83 @@ public int delete(int id) {
         }
         return (int) results.resultValue;
     }
+
+    public Map<Integer, String> getAllUserMap() {
+        SimpleResult results = TransactionWrapper.withTxn(TransactionWrapper.DB.SHARED_DB, handle -> {
+            SimpleResult dbVals = new SimpleResult();
+            List<UserDto> res = handle.attach(JdbiUser.class).getUserMap();
+            Map<Long, String> map = new HashMap<>();
+            if (res != null) {
```

Comment:
as far as I know if method of jdbi has `List` as return value and there are no results from `SQL` query it returns an empty list, maybe @Trimax can help us to clarify that, if so, we can ignore check on null.